### ISSUE TITLE
ga.grad_sqr raises AttributeError: DopFop

### ISSUE THREAD

**Issue by eric-wieser** [pygae/galgebra#182]
https://github.com/pygae/galgebra/blob/8ce4920e18ec7ddeb0a2389cb195f617950c4456/galgebra/ga.py#L1761-L1767

This code has been here since at least 7c3fd55df034e20ca2b268b9f05ca9a192713597 (the furthest we can see back into history)

https://github.com/pygae/galgebra/blob/7c3fd55df034e20ca2b268b9f05ca9a192713597/ga.py#L1377-L1383

`DopFop` has no other occurences in any file in past commit.

**Comment by utensil** [comment]
We deal with such a situation like #408. I propose that we remove the code directly and mark the related issue with a common label so we can easily find it again and forget about it. I don't want to continue to wonder what happened in history and can't do anything about it.

**Comment by eric-wieser** [comment]
Makes sense to me. Maybe we should paste a permalink to the function body in the issue pointing to the last revision before it was removed.
